### PR TITLE
ncu-team: new tool to sync members of a Github team

### PR DESCRIPTION
**Pull request by joyeecheung**

### CODE REVIEW

File: lib/team_info.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+      const info = new TeamInfo(cli, request, org, team);
+      const teamData = await info.getMemberContacts();
+      const opening = `<!-- ncu-team-sync.team(${org}/${team}) -->`;
+      const ending = `<!-- ncu-team-sync end -->`;
+      blocks.set(mapKey, `${opening}\n\n${teamData}\n\n${ending}`);
+    };
+  }
+
+  if (blocks.size === 0) {
+    throw new Error(`Could not find block matching ${RE}`);
```
Comment (cPhost) [inline lib/team_info.js]: Can we instead log out the comments instead here i guess the regular expression would be kind of daunting to log out to user?

File: lib/queries/Team.gql [hunk 2]
```diff
@@ -0,0 +1,17 @@
+query team($org: String!, $team: String!) {
+  organization(login: $org) {
+    login
+    team(slug: $team) {
+      name,
+      description,
+      members(first: 100) {
```
Comment (cPhost) [inline lib/queries/Team.gql]: Does this limit how many team members we receive?

Comment (joyeecheung) [reply lib/queries/Team.gql]: @cPhost Yeah, I plan to fix that in a later PR

Comment (cPhost) [reply lib/queries/Team.gql]: Cool! then :)

**Pull request merged**